apps/verticalhome/js/stores/item.js

```diff
@@ -360,15 +474,6 @@
           if (thisItem.type === 'app') {
             var itemObj = this.applicationSource.mapToApp(thisItem);
             addIfUnique.call(this, itemObj);
-          } else if (thisItem.type === 'divider') {
-            var divider = new GaiaGrid.Divider(thisItem);
-            this._allItems.push(divider);
-          } else if (thisItem.type === 'bookmark') {
-            var bookmark = new GaiaGrid.Bookmark(thisItem);
-            addIfUnique.call(this, bookmark);
-          } else if (thisItem.type === 'collection') {
-            var collection = new GaiaGrid.Collection(thisItem);
-            addIfUnique.call(this, collection);
```

Comment:
Looks like you can remove the Divider, Bookmark and Collection classes, given that you're removed this code.